src/init.sql

```diff
@@ -0,0 +1,202 @@
+-- This script was generated by the ERD tool in pgAdmin 4.
+-- Please log an issue at https://github.com/pgadmin-org/pgadmin4/issues/new/choose if you find any bugs, including reproduction steps.
+BEGIN;
+
+
+CREATE TABLE IF NOT EXISTS public."Events"
+(
+    "Id" serial NOT NULL,
+    "VenueId" integer NOT NULL,
+    "Name" text COLLATE pg_catalog."default" NOT NULL,
+    "Description" text COLLATE pg_catalog."default",
```

Comment:
Do we really need to explicitly define "default" settings?

Reply:
It is not necessary. The script was provided by 'pgAdmin4' tool. Currently I'm not sure why it specifies default `COLLATE` explicitly. I'll maybe dig in it later.